Google.Api.Generator/Generation/MethodDetails.cs

```diff
@@ -220,6 +220,26 @@ public ServerStreaming(ServiceDetails svc, MethodDescriptor desc) : base(svc, de
             public Typ AsyncEnumeratorTyp { get; }
         }
 
+        public sealed class ClientStreaming : MethodDetails
+        {
+            public ClientStreaming(ServiceDetails svc, MethodDescriptor desc) : base(svc, desc)
+            {
+                ApiCallTyp = Typ.Generic(typeof(ApiClientStreamingCall<,>), RequestTyp, ResponseTyp);
+                AbstractStreamTyp = Typ.Nested(svc.ClientAbstractTyp, $"{SyncMethodName}Stream");
```

Comment:
I'm assuming the build failure here is because we'll need to do a GAX release (maybe a beta) and then update the code to use that?